docs/visualize.md

```diff
@@ -0,0 +1,99 @@
+# Installation
+
+To create state machine graphs you must to install `automat` with the graphing dependencies.
+
+```
+pip install automat[visualize]
+```
+
+# Example
+
+Given a simple project file structure of:
+
+```
+.
+└── mystatemachine
+    └── __init__.py
```

Comment:
Can we move the state machine to a submodule?  One way to do this would be to rename `mystatemachine` to `mystate`

```
── mystate
     ├── __init__.py
     └── machine.py
```

This would make automat's traversal logic clearer.